sql/Register.sql

```diff
@@ -0,0 +1,12 @@
+create table user_table(
+	username varchar(30)  unique not null,
+	userpassword varchar(30) not null,
+	email varchar(30) unique not null,
+	mobileNo varchar(10) unique,
```

Comment:
Naming convention - mobileNo - use underscores to separate two words. ( e.g: mobile_no)

Reply:
Naming convention is done.